.gdbinit.md

```diff
@@ -0,0 +1,112 @@
+# set_ts
```

Comment:
Can these commands be tested using CI?

Reply:
Not trivially with our testing test up. And even if so, this isn't worth the CI time, it's just a debugging tool and a bug is not important.